content/Software Support/Upload/If-Arduino-IDE-detects-a-different-board-than-the-one-youve-connected.md

```diff
@@ -39,6 +40,23 @@ If the Reset symbol is not present, click "Select other board and port" at the b
 
 * If you're using an **Arduino Nano ESP32**, but it's being detected as a generic ESP32 board, the Arduino bootloader may be missing. To solve the issue, [reset the default Arduino bootloader](https://support.arduino.cc/hc/en-us/articles/9810414060188-Reset-the-Arduino-bootloader-on-the-Nano-ESP32).
 
-* If you're using an **Arduino UNO R4 WiFi**, but it's being detected as an ESP32 board, the USB bridge firmware may be missing. To solve the issue, follow [these instructions to restore the firmware using espflash](https://support.arduino.cc/hc/en-us/articles/9670986058780-Update-the-connectivity-module-firmware-on-UNO-R4-WiFi#espflash).
+* If you're using an **Arduino UNO R4 WiFi**, but it's being detected as an ESP32 board, the USB bridge firmware may be missing. To solve the issue, follow [these instructions to restore the firmware using espflash](https://support.arduino.cc/hc/en-us/articles/16379769332892-Restore-the-USB-connectivity-firmware-on-UNO-R4-WiFi-with-espflash).
+
+---
+
+<a id="misidentified-uno-mega-nano"></a>
+
+## If Arduino UNO (Rev3 or earlier), Mega, or classic Nano is misidentified
+
+* If your **UNO (Rev3 or earlier)** or **Mega** board is recognized as any of the following, it may indicate that the USB-to-Serial firmware is missing or corrupted:
+
+  * Unknown Device
+  * Composite Device
+  * ATmega16u2 DFU
+  * Atmega8U2 DFU
+
+  To solve the issue, [flash the USB-to-serial firmware](https://support.arduino.cc/hc/en-us/articles/4408887452434-Flash-the-USB-to-serial-firmware-for-UNO-Rev3-and-earlier-and-Mega-boards).
+
+* If you're using a **classic Nano**, but it's being detected as "USB Serial Port" <!-- on Windows --> or another generic name, the FTDI drivers may be missing. To solve the issue, follow [install the FTDI drivers](https://support.arduino.cc/hc/en-us/articles/4411305694610-Install-or-update-FTDI-drivers).
```

Comment:
In this situation, the boards would not appear in Arduino IDE.

I would distinguish between:

* Misidentified or not detected by Arduino IDE
* Misidentified or not detected by the computer

Unlike the UNO R4 WiFi and Nano ESP32, it's very unlikely that the ATmega16u2/u2 chip would end up in a state were it is identified as a different board by Arduino development tools.